src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "API Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An API Controller with or without read/write actions",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.ApiController",
+  "precedence": "9799",
```

Comment:
Any reason why 9799 and not like 9800 like the other ones in dotnet/aspnetcore?

Reply:
I think I was trying to give it an order, this I was hoping a PM would have an opinion on. We have a meeting about this I'll add you to if you have time thursday

Reply:
see [this PR](https://github.com/dotnet/aspnetcore/pull/45979#discussion_r1065265430) where doug/will settled on 9800 for 8.0

Reply:
changing all to 9800 for now.